Makefile

```diff
@@ -16,6 +16,32 @@ GO=CGO_ENABLED=0 GOOS=$(GOOS) GOARCH=$(GOARCH) GO111MODULE=on go
 GOVERSION = $(shell $(GO) version | cut -c 14- | cut -d' ' -f1)
 GOFLAGS ?=
 
+ACCOUNT_CRD_REF=master
+
+export SHREDDER_NAMESPACE=aws-account-shredder
+SHRED_ACCOUNT_CR_NAME=aws-shredder-account-delete
+ACCOUNT_OPERATOR_NAMESPACE=aws-account-operator
+
+IMAGE=$(IMAGE_REGISTRY)/$(IMAGE_REPOSITORY)/$(IMAGE_NAME)
+IMAGE_TAG=latest
+
+.PHONY: check-shred-account-id
+check-shred-account-id:
+ifndef ACCOUNT_ID_TO_SHRED
+	$(error ACCOUNT_ID_TO_SHRED is undefined)
+endif
+
+.PHONY: check-aws-account-credentials
+check-aws-account-credentials: ## Check if AWS Account Env vars are set
+ifndef AWS_ACCESS_KEY_ID
+	$(error AWS_ACCESS_KEY_ID is undefined)
+endif
+ifndef AWS_SECRET_ACCESS_KEY
+	$(error AWS_SECRET_ACCESS_KEY is undefined)
+endif
+
+only-local-ctx:
```

Comment:
I would leave a small comment above this indicating that we try enforce to only run-local

Reply:
Done